GameData/RP-0/Avionics.cfg

```diff
@@ -347,3 +347,51 @@
 		massLimit = 150.0
 	}
 }
+
+// Config for RN Proton (Third Stage)
+@PART[rn_protonk_stg3]:FOR[RP-0]
+{
+	MODULE
+	{
+		name = ModuleAvionics
+		massLimit = 700.0
+	}
+}
+// Config for RN Proton (Block D-M2)
+@PART[rn_proton_blockd]:FOR[RP-0]
+{
+	MODULE
+	{
+		name = ModuleAvionics
+		massLimit = 20.0
```

Comment:
I was only looking for Block D in combination with Proton. 20t is the max payload of the Proton. Block D is part of the Payload. so it is correct for this usage.